datahub/dnb_match/migrations/0004_delete_metadata.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.9 on 2020-01-13 10:54
+
+from django.db import migrations
+
+from datahub.core.migration_utils import DeleteModelWithMetadata
+
+
+def forward_func(apps, schema_editor):
+    """
+    Remove the rows associated with dnb_match in:
+      - django_content_type
+      - auth_permission
+    """
+    DeleteModelWithMetadata.delete_metadata(
+        apps,
+        'dnb_match',
+        'dnbmatchingcsvrecord',
+    )
+    DeleteModelWithMetadata.delete_metadata(
+        apps,
+        'dnb_match',
+        'dnbmatchingresult',
+    )
+
+
+def reverse_func(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dnb_match', '0003_delete'),
+    ]
+
+    operations = [
+        migrations.RunPython(forward_func, reverse_func)
```

Comment:
Could use `migrations.RunPython.noop` instead of `reverse_func`:

```suggestion
        migrations.RunPython(forward_func, migrations.RunPython.noop)
```

(Also I'd say this should technically have `elidable=True`, since we wouldn't want this if the migrations were squashed, but it doesn't matter if we're going to delete the app anyway.)